lib/app.js

```diff
@@ -2,15 +2,18 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
```

Comment:
In terms of use it wouldn't differ much. But it allows us to not hog in another library / extra code to handle the withRoute case which I'm pretty sure not everyone is going to use 👍